bots/psuteam7bot/movement.js

```diff
@@ -0,0 +1,408 @@
+const movement = {}
+
+//Array for getting direction after rotation
+movement.directions = [{ x: 0, y: -1 }, { x: 1, y: -1 }, { x: -1, y: -1 }, { x: 1, y: 0 }, { x: -1, y: 0 }, { x: 0, y: 1 }, { x: 1, y: 1 }, { x: -1, y: 1 }]
```

Comment:
Think these are out of order. The `rotateDirection` method is wonky for me and I think this might be some of the reason.

Reply:
To clarify, I imagine these have to be in clockwise/counter-clockwise order. Going from `(0, -1)` to `(1, -1)` is fine, but next should be `(1, 0)` and not `(-1, -1)`